testing/helm/tests/resourced.yaml

```diff
@@ -510,6 +510,16 @@ sasha:
     resources:
       cpu: "1"
       memory: "4Gi"
+      
+  debug-housekeeping:
+    #disabled: true
+    #interval: 1
+    #at: "* 3 * * *"
+    #expiryDefault: 7
+    #user: sasha
+    resources:
+      cpu: "1"
+      memory: "4Gi"
```

Comment:
this service in particular should require a trivial amount of memory, to emphasis that, let's reduce this to 1Gi